config/locales/server.en.yml

```diff
@@ -4373,7 +4373,7 @@ en:
         This badge is granted when you use all %{max_likes_per_day} of your daily likes for 5 days. Thanks for taking the time actively encouraging the best conversations every day!
     crazy_in_love:
       name: Crazy in Love
-      description: Used 50 likes in a day 20 times
+      description: Used %{max_likes_per_day} likes in a day 20 times
```

Comment:
Where is this translation called from? Don't we have to pass in the `max_likes_per_day` when fetching the translation?

Reply:
Hi @tgxworld, I'm new to this codebase. I naively assumed that they all inherit from the English locale. Upon checking, this has been hardcoded on the other locales as well. For example, the es and it locales have it too

https://github.com/discourse/discourse/blob/16137308b011146be29f88c3856b154d55d04abf/config/locales/server.es.yml#L3941

https://github.com/discourse/discourse/blob/16137308b011146be29f88c3856b154d55d04abf/config/locales/server.it.yml#L3464

I'll have to read up more on how translations are done here.

Reply:
Got this response on Discourse

https://meta.discourse.org/t/crazy-in-love-badge-description-not-reflecting-max-likes-per-day-setting/151127/2?u=rkda

It appears this will be the reference for all the other locales.